host_vars/gcp_mariadb_dev1.yml

```diff
@@ -0,0 +1,2 @@
+---
+gcp_mariadb_dev: 10.0.10.6
```

Comment:
In order to make sure this PR doesn't break things. I am testing this on Google Cloud to lower the surprise factor. I am able to create a db and a db user. It could have been anywhere but I want to be sure that my new PR's will not break our current implementation.